recipe/build.sh

```diff
@@ -55,13 +55,3 @@ if [[ "$CONDA_BUILD_CROSS_COMPILATION" != "1" ]]; then
   fi
 fi
 make install
-
-
-if [[ "${target_platform}" == osx-* ]]; then
-    for lib in blas cblas lapack lapacke; do
-        mv $PREFIX/lib/lib$lib.dylib $PREFIX/lib/lib$lib.$PKG_VERSION.dylib
-        install_name_tool -id $PREFIX/lib/lib$lib.3.dylib $PREFIX/lib/lib$lib.$PKG_VERSION.dylib
-        ln -s  $PREFIX/lib/lib$lib.$PKG_VERSION.dylib $PREFIX/lib/lib$lib.dylib
-        ln -s  $PREFIX/lib/lib$lib.$PKG_VERSION.dylib $PREFIX/lib/lib$lib.3.dylib
-    done
-fi
```

Comment:
Not sure why this + patch 0001 was necessary at the time (https://github.com/conda-forge/lapack-feedstock/commit/2fe8a252a5fd198c1b17da9058a0db2671660c52), but everything seems to point to the fully-versioned lib correctly even without it:
```
?rwxrwxr-x 0/0     388208 2024-11-07 09:30:45 lib/libblas.3.9.0.dylib 
?rwxrwxr-x 0/0          0 2024-11-07 09:30:45 lib/libblas.3.dylib -> libblas.3.9.0.dylib 
?rwxrwxr-x 0/0          0 2024-11-07 09:30:45 lib/libblas.dylib -> libblas.3.9.0.dylib 
```

Reply:
The patch is about avoiding setting current_version and compatibility_version in libblas.dylib so that downstream consumers don't record it to facilitate the switching of implementations.

Reply:
Thanks. I reinstated patch 0001 and put that detail in the commit message